upgrade-2.x.sh

```diff
@@ -976,7 +982,7 @@ if ! version_gt "$VERSION" "$preferred_hostos_version" &&
             download_uri=https://github.com/resin-os/resinhup/raw/master/upgrade-binaries/$binary_type
             for binary in $tools_binaries; do
                 log "Installing $binary..."
-                curl --retry 10 -f -s -L -o $tools_path/$binary $download_uri/$binary || log ERROR "Couldn't download tool from $download_uri/$binary, aborting."
+                CURL_CA_BUNDLE=${TMPCRT} curl --retry 10 -f -s -L -o $tools_path/$binary $download_uri/$binary || log ERROR "Couldn't download tool from $download_uri/$binary, aborting."
```

Comment:
Here and up ahead, should this line have dis-ambiguating curly braces around the variables?  Ie:

```
${tools_path}/${binary} ${download_uri}/${binary}
```

Reply:
I don't like changing things like this out-of-scope, but I agree it could be standardized. There's no real difference unless you're concatenating strings (i.e. `${a_string}another_string${a_third_string}` != `$a_stringanother_string$a_third_string`), so I'd prefer to leave it for now and clean all those up in one fell swoop.